suave/devenv/blockscout/envs/common-blockscout.env

```diff
@@ -0,0 +1,239 @@
+# DOCKER_TAG=
+ETHEREUM_JSONRPC_VARIANT=geth
+ETHEREUM_JSONRPC_HTTP_URL=http://host.docker.internal:8545/
+# ETHEREUM_JSONRPC_FALLBACK_HTTP_URL=
+DATABASE_URL=postgresql://blockscout:ceWb1MeLBEeOIfk65gU8EjF8@blockscout-db:5432/blockscout?ssl=false
+ETHEREUM_JSONRPC_TRACE_URL=http://host.docker.internal:8545/
+# ETHEREUM_JSONRPC_FALLBACK_TRACE_URL=
+# ETHEREUM_JSONRPC_HTTP_TIMEOUT=
+CHAIN_TYPE=suave
+CHAIN_ID=1337
+NETWORK=
+SUBNETWORK=Awesome chain
+LOGO=/images/blockscout_logo.svg
+ETHEREUM_JSONRPC_WS_URL=ws://host.docker.internal:8546/
```

Comment:
**issue:**

for some reason this seems to keep failing:

<img width="675" alt="image" src="https://github.com/flashbots/suave-geth/assets/83900239/00a43785-ae97-46c3-ac59-75757d2c36d3">

```
websocat "ws://localhost:8080/socket/v2/websocket?vsn=2.0.0" -vvv
[INFO  websocat::lints] Auto-inserting the line mode
[DEBUG websocat] Done third phase of interpreting options.
[DEBUG websocat] Done fourth phase of interpreting options.
[DEBUG websocat] Preparation done. Now actually starting.
[DEBUG websocat::sessionserve] Serving Line2Message(ThreadedStdio) to Message2Line(WsClient("ws://localhost:8080/socket/v2/websocket?vsn=2.0.0")) with Options { websocket_text_mode: true, websocket_protocol: None, websocket_reply_protocol: None, udp_oneshot_mode: false, udp_broadcast: false, udp_multicast_loop: false, udp_ttl: None, udp_join_multicast_addr: [], udp_join_multicast_iface_v4: [], udp_join_multicast_iface_v6: [], udp_reuseaddr: false, unidirectional: false, unidirectional_reverse: false, max_messages: None, max_messages_rev: None, exit_on_eof: false, oneshot: false, unlink_unix_socket: false, unix_socket_accept_from_fd: false, exec_args: [], ws_c_uri: "ws://0.0.0.0/", linemode_strip_newlines: false, linemode_strict: false, origin: None, custom_headers: [], custom_reply_headers: [], websocket_version: None, websocket_dont_close: false, websocket_ignore_zeromsg: false, one_message: false, no_auto_linemode: false, buffer_size: 65536, broadcast_queue_len: 16, read_debt_handling: Warn, linemode_zero_terminated: false, restrict_uri: None, serve_static_files: [], exec_set_env: false, no_exit_on_zeromsg: false, reuser_send_zero_msg_on_disconnect: false, process_zero_sighup: false, process_exit_sighup: false, process_exit_on_disconnect: false, socks_destination: None, auto_socks5: None, socks5_bind_script: None, tls_domain: None, tls_insecure: false, headers_to_env: [], max_parallel_conns: None, ws_ping_interval: None, ws_ping_timeout: None, request_uri: None, request_method: None, request_headers: [], autoreconnect_delay_millis: 20, ws_text_prefix: None, ws_binary_prefix: None, ws_binary_base64: false, ws_text_base64: false, close_status_code: None, close_reason: None, asyncstdio: false, foreachmsg_wait_reads: false, announce_listens: false, timestamp_monotonic: false, print_ping_rtts: false, byte_to_exit_on: 28, max_ws_message_length: 209715200, max_ws_frame_length: 104857600, preamble: [], preamble_reverse: [], compress_deflate: false, compress_zlib: false, compress_gzip: false, uncompress_deflate: false, uncompress_zlib: false, uncompress_gzip: false, jsonrpc_omit_jsonrpc: false, inhibit_pongs: None, max_sent_pings: None }
[INFO  websocat::stdio_threaded_peer] get_stdio_peer (threaded)
[DEBUG websocat::sessionserve] Underlying connection established
[INFO  websocat::ws_client_peer] get_ws_client_peer
websocat: WebSocketError: I/O failure
websocat: error running
```

do you have any idea why?

Reply:
also:

```
application=phoenix [info] CONNECTED TO BlockScoutWeb.UserSocketV2 in 1ms                                                                                                             │
│   Transport: :websocket                                                                                                                                                                                       │
│   Serializer: Phoenix.Socket.V1.JSONSerializer                                                                                                                                                                │
│   Parameters: %{}                                                                                                                                                                                             │
│ 2023-10-20T12:10:08.667 [error] Ranch listener BlockScoutWeb.Endpoint.HTTP had connection process started with :cowboy_clear:start_link/4 at #PID<0.14257.53> exit with reason: {:function_clause, [{Phoenix. │
│ Socket.Message, :from_map!, [["47", "47", "transactions:new_transaction", "phx_join", %{}]], []}, {Phoenix.Socket, :__in__, 2, [file: 'lib/phoenix/socket.ex', line: 466]}, {Phoenix.Endpoint.Cowboy2Handler, │
│  :websocket_handle, 2, [file: 'lib/phoenix/endpoint/cowboy2_handler.ex', line: 175]}, {:cowboy_websocket, :handler_call, 6, [file: '/app/deps/cowboy/src/cowboy_websocket.erl', line: 528]}, {:cowboy_http, : │
│ loop, 1, [file: '/app/deps/cowboy/src/cowboy_http.erl', line: 257]}, {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 226]}]}
```